include/rabbit_stomp.hrl

```diff
@@ -21,3 +21,31 @@
                               ssl_cert_login}).
 
 -define(SUPPORTED_VERSIONS, ["1.0", "1.1", "1.2"]).
+
+-define(INFO_ITEMS,
+        [conn_name,
+         connection,
+         connection_state,
+         session_id,
+         channel,
+         version,
+         ssl_cert_login,
+         implicit_connect,
+         auth_login,
+         auth_mechanism,
+         ssl_login_name,
```

Comment:
`ssl_login_name` and `ssl_cert_login` should be covered by `auth_login` and `auth_mechanism`